Basic Input Output Problems/best_index.cpp

```diff
@@ -0,0 +1,43 @@
+#include<iostream>
```

Comment:
Please fix your code so that HackerEarth's compiler doesn't show any warnings as it's showing now as _Time Limit Exceeded_.